1657389 Add descriptions for SEO (series of related PRs) (#19589)
dotnet/docs: docs/csharp/programming-guide/concepts/async/start-multiple-async-tasks-and-process-them-as-they-complete.md

---
title: Process asynchronous tasks as they complete

> description: This example shows how to use Task.WhenAny in C# to start multiple tasks at the same time and process them as they finish, rather than process them in order.

Reviewer: adding a clarification:

```suggestion
description: This example shows how to use Task.WhenAny in C# to start multiple tasks and process their results as they finish, rather than process them in the order started.
```
